src/widgets/label/lv_label.c

```diff
@@ -891,6 +891,27 @@ static void draw_main(lv_event_t * e)
     draw_ctx->clip_area = clip_area_ori;
 }
 
+static void overwrite_anim_property(lv_anim_t * dest, const lv_anim_t * src, lv_label_long_mode_t mode)
+{
+    switch(mode) {
+        case LV_LABEL_LONG_SCROLL:
+            dest->act_time = src->act_time;
```

Comment:
I think `act_time` shouldn't be overwritten. I think it's not useful and LVGL calculates it to keep the current scroll position.

Reply:
Sounds great!